### PR TITLE
Add Modding Hubs Discord server articles

### PR DESCRIPTION
**Pull request by Hiviexd**
Related to #6904 

also added the discord servers page to `Community` because it was pretty inaccessible before.

also these infoboxes are pretty cool

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

**Comment by Hiviexd** [comment]
about translations, I thought about outdating them but for such a simple change, maybe it's better to handle them in this PR?
didn't want to mess with that initially as I can only translate `fr.md` and `de.md` semi-confidently

**Comment by Kotoki1337** [comment]
`zh.md`: Just realized there's no simplified version.
`zh-tw.md`: `社群 Discord 伺服器`

**Comment by Ganondorf-1** [comment]
French: `Serveurs Discord communautaires`

**Comment by hikikuman** [comment]
Indonesian: `Server Discord komunitas`

**Comment by Hiviexd** [comment]
thanks guys 👍
also translated `de.md` and `uk.md` thanks to native speaking friends

**Comment by minusQuantumNeko** [comment]
Better version for `uk.md`: `Discord сервери спiльноти`
`ru.md`: `Discord сервера сообщества`

**Comment by The-Last-Cookie** [comment]
My suggestion for German is `Discord-Server von der Community` if the `osu!dev Discord server` article will not be moved in this category.

Using either `Community Discord-Server` or `Community-Discord-Server` looks less readable to me.

I would like to know what your native friend(s) think about this @Hiviexd

**Comment by Hiviexd** [comment]
@The-Last-Cookie they said `Discord-Server von der Community` feels too wordy and prefer current one

**Review by minusQuantumNeko — APPROVED** [review]

**Review by TicClick — APPROVED** [review]
